cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -95,9 +95,10 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
             cy.window().then(win => {
               const media = getBlockData('video', win.SIMORGH_DATA.pageData);
               if (media && media.type === 'video') {
-                cy.get(
-                  'div[class^="StyledVideoContainer"] img[class^="StyledImg"]',
-                )
+                cy.get('div[class^="StyledVideoContainer"]')
+                  .within(() => {
+                    cy.get('button');
+                  })
```

Comment:
- This change alters the assertion this test makes.
- This test shouldn't need to interact with a `<button>`.
- Without these changes, does the test fail?

Reply:
yes the test fails, currently looking for an alternative implementation

Reply:
I just investigated this and it fails, because the Guidance `div` overlays the `img`, blocking the `.click()`. Play Button will always be on top and will also disappear with Placeholder, so can I suggest we tweak the selection on L98 to the following and remove the `.within()` block? This should return the test back to normal.
```
cy.get(
  'div[class^="StyledVideoContainer"] button[class*="StyledPlayButton"]',
)
```